.changeset/twenty-camels-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Security: fix for `semver` vulnerability
```

Comment:
[note] Our maintainer guide now doesn't mention the `Security:` prefix. This doesn't block this PR, but I'm a bit curious. 😅 

https://github.com/stylelint/stylelint/blob/a42f95582c2a928122fc1e59f9f8acc078fb9b91/docs/maintainer-guide/pull-requests.md?plain=1#L22-L23

Reply:
It's an oversight. It should list all the ones from https://keepachangelog.com/en/1.0.0/. I'll open a PR.